yamls/gstreamer/startup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
why are there scripts in the yaml folder? make a new folder if you need scripts or untrack them